SpatialGDK/Source/SpatialGDK/Private/EngineClasses/Components/SpatialPingComponent.cpp

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "EngineClasses/Components/SpatialPingComponent.h"
+
+#include "GameFramework/PlayerController.h"
+#include "Net/UnrealNetwork.h"
+#include "TimerManager.h"
+#include "Engine/World.h"
+
+USpatialPingComponent::USpatialPingComponent(const FObjectInitializer& ObjectInitializer)
+	: Super(ObjectInitializer)
+{
+	PrimaryComponentTick.bCanEverTick = false;
+	PrimaryComponentTick.bStartWithTickEnabled = false;
+
+	bReplicates = true;
+
+	bStartWithPingEnabled = true;
```

Comment:
How come we do this here and the header defaults to true?

Reply:
Yeah this is redundant. I'll remove it from the constructor and leave it in the header as the other defaults are already set there.